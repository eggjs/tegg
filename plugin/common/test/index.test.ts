import { expect, test } from 'vitest';

import * as exports from '../src/index.ts';

test('should export stable', async () => {
  expect(exports).toMatchSnapshot();
});
