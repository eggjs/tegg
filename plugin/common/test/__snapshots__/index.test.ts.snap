// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should export stable 1`] = `
{
  "EGG_CONTEXT": Symbol(context#eggContext),
  "ROOT_PROTO": Symbol(context#rootProto),
  "TEGG_CONTEXT": Symbol(context#teggContext),
}
`;
